pdns/rec_channel_rec.cc

```diff
@@ -990,6 +990,7 @@ void registerAllStats()
   addGetStat("edns-ping-matches", &g_stats.ednsPingMatches);
   addGetStat("edns-ping-mismatches", &g_stats.ednsPingMismatches);
   addGetStat("dnssec-queries", &g_stats.dnssecQueries);
+  addGetStat("variable-responses", &g_stats.variableResponses);
```

Comment:
This new stat value will need some documentation, and is currently not exported over `SNMP` (would require a `MIB` update as well).

Reply:
documented and SNMPd